stacks-signer/src/signer.rs

```diff
@@ -28,7 +28,9 @@ use libsigner::{
     BlockProposalSigners, BlockRejection, BlockResponse, MessageSlotID, RejectCode, SignerEvent,
     SignerMessage,
 };
+use rand_core::OsRng;
```

Comment:
Why `OsRng`?  What's wrong with `rand::RngCore` etc.?

Reply:
Just to be cautious. I'm not an expert in which random number generators are cryptographically secure or not, and `OsRng` is the one used in the existing `wsts` and Signer code in `dkg_private_begin` among other places. So I figured that it's best to be consistent and not introduce any other rng.

I can definitely make the encryption logic generic with a `rand::RngCore` trait bound, or use any other rng if you have any concerns about `OsRng` in particular.

Let me know what you think.

Reply:
Anywhere `wsts` is hard coding a specific `RNG` is a bug, please file a ticket

Reply:
Okay, will do. This will entail changes in the public interfaces though.

Reply:
https://github.com/Trust-Machines/wsts/issues/63